QuizApp/Screens/QuestionPage/Cell/AnswerCell.swift

```diff
@@ -0,0 +1,179 @@
+        NSLayoutConstraint.activate([
+            containerView.topAnchor.constraint(
+                equalTo: contentView.topAnchor,
+                constant: Constants.AnswerCellConstants.containerVerticalPadding
+            ),
+            containerView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor),
+            containerView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor),
+            containerView.bottomAnchor.constraint(
+                equalTo: contentView.bottomAnchor,
+                constant: -Constants.AnswerCellConstants.containerVerticalPadding
+            ),
+            containerView.heightAnchor.constraint(equalToConstant: Constants.AnswerCellConstants.containerHeight)
+        ])
+    }
+    
+    private func setupTitleLabelConstraints() {
+        NSLayoutConstraint.activate([
+            titleLabel.leadingAnchor.constraint(
+                equalTo: containerView.leadingAnchor,
+                constant: Constants.AnswerCellConstants.titleLabelHorizontalPadding
+            ),
+            titleLabel.trailingAnchor.constraint(
+                equalTo: scoreLabel.leadingAnchor,
+                constant: -Constants.AnswerCellConstants.titleLabelHorizontalPadding
+            ),
+            titleLabel.centerYAnchor.constraint(equalTo: containerView.centerYAnchor)
+        ])
+    }
+    
+    private func setupScoreLabelConstraints() {
+        NSLayoutConstraint.activate([
+            scoreLabel.trailingAnchor.constraint(
+                equalTo: starImageView.leadingAnchor,
+                constant: Constants.AnswerCellConstants.scoreLabelPadding
+            ),
+            scoreLabel.centerYAnchor.constraint(equalTo: containerView.centerYAnchor)
+        ])
+    }
+    
+    private func setupStarImageViewConstraints() {
+        NSLayoutConstraint.activate([
+            starImageView.trailingAnchor.constraint(
+                equalTo: containerView.trailingAnchor,
+                constant: -Constants.AnswerCellConstants.titleLabelHorizontalPadding
+            ),
+            starImageView.centerYAnchor.constraint(equalTo: containerView.centerYAnchor),
+            starImageView.heightAnchor.constraint(equalToConstant: Constants.AnswerCellConstants.starImageHeight)
+        ])
+    }
+    
+    // MARK: - Configuration
+    func configure(with title: String) {
+        titleLabel.text = title
+    }
+    
+    // MARK: - Color Changing Success/Wrong
+    func updateState(isCorrect: Bool) {
+        if isCorrect {
+            // Correct answer
+            containerView.backgroundColor = Constants.AnswerCellConstants.success
+            titleLabel.textColor = Constants.Colors.neutralWhite
+            scoreLabel.isHidden = false
+            starImageView.isHidden = false
+        } else {
+            // Wrong answer
+            containerView.backgroundColor = Constants.AnswerCellConstants.wrong
+            titleLabel.textColor = Constants.Colors.neutralWhite
```

Comment:
why do you need to set textcolor again, you are not changing the colour though

Reply:
from black color it changes to white color after you select the cell.